src/nuclio/api-gateways/new-api-gateway-wizard/new-api-gateway-wizard.component.js

```diff
@@ -93,8 +93,9 @@
          */
         function onInit() {
             var authenticationType = lodash.get(ctrl.apiGateway, 'spec.authenticationMode', null);
-            ctrl.authMode = ApiGatewaysService.getAuthModes();
-            ctrl.selectedAuthenticationType = lodash.defaultTo(authenticationType, ctrl.authMode[0].id);
+
+            ctrl.authModes = ApiGatewaysService.getAuthModes(ConfigService.nuclio.allowedAuthenticationModes);
```

Comment:
You don't need to pass `ConfigService.nuclio.allowedAuthenticationModes` from the component to the service's method. You can inject `ConfigService` inside another service